routes/cartRouter.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const router  = express.Router();
+
+const {loginRequired} = require("../utils/auth")
+const { cartController } = require('../controllers');
+
+router.post('/',loginRequired, cartController.additionCart);
+router.get('/',loginRequired, cartController.getByCart);
+router.patch('/change', loginRequired,cartController.cartQuantityChange);
```

Comment:
- REST에서 동사는 사용하지 않습니다.
- 특정 cart 1개를 수정하는 행위를 할려면 아래와 같이 path parmeter로 cart의 id를 받아서 수정하는 것이 적절해 보입니다.
```suggestion
router.patch('/:cartId', loginRequired,cartController.cartQuantityChange);
```